tests/integration/authentication/test_401_errors.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import pytest
+from lib import Assertions, Generators
+
+TOKENS = [None, "Bearer xyzcba", "Bearer", "junk"]

```

Comment:
`TOKENS` here should be moved out into constants and shared across all of the test cases we have that use them - across environments - same with `METHODS` and `CORRELATION_IDS` I think.

Reply:
Done